lib/projects/localDev/AppDevModeInterface.ts

```diff
@@ -144,24 +145,29 @@ class AppDevModeInterface {
   // @ts-expect-error TODO: reconcile types between CLI and UIE Dev Server
   // In the future, update UIE Dev Server to use LocalDevState
   async setup(args): Promise<void> {
-    if (!this.app) {
+    if (!this.appNode) {
       return;
     }
 
-    if (this.app?.config.distribution === APP_DISTRIBUTION_TYPES.MARKETPLACE) {
-      try {
-        await this.fetchMarketplaceAppData();
+    try {
+      await this.fetchAppData();
```

Comment:
The only change to logic is in the `setup` function. We now fetch app data (which is required to check installation and get the install link) and check if the app is installed for _all_ apps rather than just marketplace apps